header.php

```diff
@@ -62,11 +62,11 @@
 
 	<header id="masthead" class="site-header clearfix <?php echo esc_attr( $header_class ); ?>">
 
-		<?php if ( spacious_options( 'spacious_activate_top_header_bar', 0 ) == 1 ) { ?>
+		<?php if ( get_theme_mod( 'spacious_activate_top_header_bar', 0 ) == 1 ) { ?>
 			<div id="header-meta">
 				<div class="inner-wrap clearfix">
 					<?php
-					if ( spacious_options( 'spacious_activate_social_links', 0 ) == 1 ) {
+					if ( get_theme_mod( 'spacious_activate_social_links', 0 ) == 1 ) {
```

Comment:
Use strict comparison like here: https://github.com/themegrill/spacious/pull/62/files#diff-e7f789e333fc7ce27ae3d6999cca0b78637e8b96c734dea9aedb855b278b7eb9R156

Also, make sure it is being saved as number not string or boolean